src/core/API.cpp

```diff
@@ -209,7 +209,6 @@ sc2::UnitTypeData Observer::GetUnitTypeData(sc2::UNIT_TYPEID id_) const {
             break;
 
         case sc2::UNIT_TYPEID::TERRAN_MARAUDER:
-        case sc2::UNIT_TYPEID::TERRAN_REAPER:
             data.tech_requirement = sc2::UNIT_TYPEID::TERRAN_BARRACKSTECHLAB;
```

Comment:
Unfortunately, this patch is not enough.
You should move the reaper ability from here
https://github.com/alkurbatov/suvorov-bot/blob/cf3bdf5d5190a0d417b9f6d880c35a8df7df3d2b/src/blueprints/Blueprint.cpp#L106
to here:
https://github.com/alkurbatov/suvorov-bot/blob/cf3bdf5d5190a0d417b9f6d880c35a8df7df3d2b/src/blueprints/Blueprint.cpp#L102

Reply:
This has been done in commit 250f56d and I've reverted the #43 commit, so hopefully this should be good to accept now.

Reply:
Thank you!
Currently we have a lot of intermediate changes in the history. Please squash the commits and I'll merge the request.

Reply:
Squashed and force-pushed. :)